index.d.ts

```diff
@@ -0,0 +1 @@
+export default function lowercaseKeys(input: object): object;
```

Comment:
Is there any way to preserve the value types here? So if the input object has all values as `string`, the returned object would be typed as such too.

Reply:
This should be generic.
https://www.typescriptlang.org/docs/handbook/generics.html
```ts
export default function lowercaseKeys<T>(input: { [k: string]: T }): { [k: string]: T };
```